care/facility/models/facility.py

```diff
@@ -297,6 +307,33 @@ def get_features_display(self):
     CSV_MAKE_PRETTY = {"facility_type": (lambda x: REVERSE_FACILITY_TYPES[x])}
 
 
+class FacilityHubSpoke(BaseModel, FacilityRelatedPermissionMixin):
```

Comment:
Remove `FacilityRelatedPermissionMixin` as it does not make sense without a `facility` attribute